modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml

```diff
@@ -6395,6 +6395,40 @@ paths:
                                     $ref: "#/components/schemas/DocumentShortcut"
                                 type: array
             tags: ["DocumentShortcut"]
+    "/asset-libraries/{assetLibraryId}/document-shortcuts/{documentId}":
```

Comment:
Hi @jonathanmccann, unsure, but I think we can avoid this endpoint and put the post method directly in 

`/asset-libraries/{assetLibraryId}/document-shortcuts`

Reply:
I tried that but then the required `documentId` was not passed through. Since both parameters are required they need to be defined in the URL. See also https://github.com/liferay/liferay-portal/blob/master/modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml#L8933

If I'm misunderstanding and there's a way to have them both passed through please let me know.

Reply:
Hi @jonathanmccann, I understand now why I found this confusing. This is a bit different use case of what we normally do. Since the end point is for posting a shortcut, normally we would put the documentID as a property, not as a  path parameter.

I was checking the story and that is how it was designed, so I would double check with Austin